openmp-helloworld/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+project(openmp_helloworld)
+
+cmake_minimum_required(VERSION 3.16)
+
+# Search for rocm in common locations
+list(APPEND CMAKE_PREFIX_PATH /opt/rocm/hip /opt/rocm)
+
+# Find HIP
+find_package(hip REQUIRED)
+
+# Find OpenMP
+find_package(openmp REQUIRED)
```

Comment:
This step is broken, the openmp-extra package is providing the omp.h, libomp.so, but missing the Findopenmp.cmake file. 
```
CMake Error at CMakeLists.txt:10 (find_package):
  By not providing "Findopenmp.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "openmp", but
  CMake did not find one.

  Could not find a package configuration file provided by "openmp" with any
  of the following names:

    openmpConfig.cmake
    openmp-config.cmake

  Add the installation prefix of "openmp" to CMAKE_PREFIX_PATH or set
  "openmp_DIR" to a directory containing one of the above files.  If "openmp"
  provides a separate development package or SDK, be sure it has been
  installed.
```

Reply:
It should be `find_package(OpenMP REQUIRED)`

Reply:
Thanks! That fixed this issue.